core/tests/protractor_utils/editor.js

```diff
@@ -142,6 +142,20 @@ var expectContentTextToEqual = function(text) {
 // Additional arguments may be sent to this function, and they will be
 // passed on to the relevant interaction editor.
 var setInteraction = function(interactionId) {
```

Comment:
Please could you add a comment here that this is to be used as the standard way to specify interactions for most purposes.